docs/pages/versions/unversioned/sdk/background-task.mdx

```diff
@@ -179,6 +181,79 @@ Since the Background Tasks API on iOS and the WorkManager API on Android limit t
 
 The last registered background task determines the minimum interval for execution.
 
+## Testing Background Tasks
+
+Background tasks can be tested using the [`triggerTaskWorkerForTestingAsync`](#backgroundtasktriggertaskworkerfortestingasync) method. This method will invoke the `BGTaskScheduler` on iOS and run all registered tasks directly on Android. This is useful for testing the behavior of your background tasks without having to wait for the system to trigger them.
+
+This method is only available in development mode. It will not work in production builds.
+
+```tsx
+import * as BackgroundTask from 'expo-background-task';
+import { Button } from 'react-native';
+
+function App() {
+  const triggerTask = async () => {
+    await BackgroundTask.triggerTaskWorkerForTestingAsync();
+  };
+
+  return <Button title="Trigger Background Task" onPress={triggerTask} />;
+}
+```
+
+## Inspecting background tasks&ensp;<PlatformTags platforms={['android']} />
+
+To troubleshoot or debug issues with background tasks on Android, use the `adb` tool included with the Android SDK to inspect scheduled tasks:
+
+```bash
+adb shell dumpsys jobscheduler | grep -A 40 -m 1 <package-name>
+```
+
+The output from this command will show you the scheduled tasks for your app, including their status, constraints, and other information. Look for the `JOB` line to find the ID of the job and other details in the output:
+
+```bash
+  JOB #u0a453/275: 216a359 <package-name>/androidx.work.impl.background.systemjob.SystemJobService
+    u0a453 tag=*job*/<package-name>/androidx.work.impl.background.systemjob.SystemJobService#275
+    Source: uid=u0a453 user=0 pkg=<package-name>
+    ...
+    Required constraints: TIMING_DELAY CONNECTIVITY UID_NOT_RESTRICTED [0x90100000]
+    Preferred constraints:
+    Dynamic constraints:
+    Satisfied constraints: CONNECTIVITY DEVICE_NOT_DOZING BACKGROUND_NOT_RESTRICTED TARE_WEALTH WITHIN_QUOTA UID_NOT_RESTRICTED [0x1b500000]
+    Unsatisfied constraints: TIMING_DELAY [0x80000000]
+    ...
+    Enqueue time: -8m12s280ms
+    Run time: earliest=+6m47s715ms, latest=none, original latest=none
+    Restricted due to: none.
+    Ready: false (job=false user=true !restricted=true !pending=true !active=true !backingup=true comp=true)
+
+```
+
+The first line contains the Job ID (275). The `Run time: earliest` value indicates the earliest time the task may start, while `enqueue time` shows how long ago the task was scheduled.
+
+To force a task to run, use the `adb shell am broadcast` command:
```

Comment:
```suggestion
To force a task to run, use the `adb shell am broadcast` command. Move your app to the background before running this command, as the task will not run if the app is in the foreground.
```

Lets add this context before running the command since the callout can be missed because it is after the command is presented.